utils/add_platform_rule.py

```diff
@@ -180,8 +211,32 @@ def which(program):
     return None
 
 
+def createNodeRuleFunc(args):
+    group_path = os.path.join(OCP_RULE_DIR, args.group)
+    if args.group:
+        if not os.path.isdir(group_path):
+            print("ERROR: The specified group '%s' doesn't exist in the '%s' directory" % (
+                args.group, OCP_RULE_DIR))
+            return 0
+
+    rule_path = os.path.join(group_path, args.rule)
+    rule_yaml_path = os.path.join(rule_path, 'rule.yml')
+
+    mkdir_p(rule_path)
+    with open(rule_yaml_path, 'w') as f:
+        var_list = [ f"    {k.strip()}" for k in args.template_vars.split(",")]
```

Comment:
An alternative to formatting this way would be to put everything in a dictionary, then render the YAML using a library (like ruamel, or pyyaml).

Reply:
I agree that the current approach is not ideal, I merely followed the existing one.

I can update the script to use a more yaml friendly way.